### DISCUSSION TITLE
It seems that my private contributions have not been counted in the card

### DISCUSSION THREAD

**Discussion by KJZH001** [anuraghazra/github-readme-stats#2959] [Q&A] [answered]
I have checked the Readme (Simplified Chinese version), but it seems that I couldn't find the reason, so I would like to seek some help.

https://github-readme-stats.vercel.app/api?username=KJZH001&count_private=true&show_icons=true&line_height=20

This is the link I used, and you can also find it in my GitHub Readme.

Thanks!

**Comment by rickstaa** [comment, marked as answer]
Duplicate of https://github.com/anuraghazra/github-readme-stats/discussions/2956.

@KJZH001 Thanks for your question. It looks like the card is working correctly. You can check this yourself in the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer).

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=KJZH001)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=KJZH001)](https://github.com/anuraghazra/github-readme-stats)
```

![image](https://github.com/anuraghazra/github-readme-stats/assets/17570430/0214aa6f-ea95-44f8-aaa9-20e87c8ea69b)

We removed the `count_private` option since it counted Contributions instead of commits (see https://github.com/anuraghazra/github-readme-stats/pull/2736, https://github.com/anuraghazra/github-readme-stats/discussions/1679 and https://github.com/anuraghazra/github-readme-stats/issues/2517). If you want your private commits to appear, please [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own).

**Comment by KJZH001** [comment]
@rickstaa Thank you for your reply.

It seems I must deploy my own if I want count my private contributions,Right?

So,I think the Simplified Chinese version README is out of date,isn't it?

You can see here
---

将私人项目贡献添加到总提交计数中
你可以使用参数 ?count_private=true 把私人贡献计数添加到总提交计数中。

注：如果你是自己部署本项目，私人贡献将会默认被计数，**如果不是自己部署，你需要分享你的私人贡献计数。**

选项: &count_private=true

---

**Reply by rickstaa** [reply]
@KJZH001 That is correct; most translations are outdated. We are in the process of migrating to automatic translations (see https://github.com/anuraghazra/github-readme-stats/issues/2053 and https://github.com/anuraghazra/github-readme-stats/issues/2637). Feel free to accept my answer above if your issue is solved after deploying your instance.

**Comment by KJZH001** [comment]
Okay, I've self-built and successfully solved the problem of private contributions not being counted

I have an additional question though, has the algorithm for calculating rank ever changed? Before I made some of my repositories private my rank was seen as A, now even with private contributions it's only a B. Of course it's not a big deal, I'm just asking out of curiosity!

[![info](https://github-readme-stats-gray-alpha-29.vercel.app/api?username=KJZH001&count_private=true&show_icons=true&line_height=20 )](https://github.com/anuraghazra/github-readme-stats)

BTW your answer I have marked as an answer

**Reply by rickstaa** [reply]
@KJZH001 great to hear that your problem is solved. The ranking, indeed, has changed. In the old ranking, achieving a rank higher than A or lower than B was complex. For more information, see https://github.com/anuraghazra/github-readme-stats/issues/2265). The ranking will likely change one more time later this month since we were able to retrieve data about our user base (see https://github.com/anuraghazra/github-readme-stats/pull/2962).

**Reply by KJZH001** [reply]
Ok,I got it.Thanks for your help!

**Comment by AlessioQuercia** [comment]
Hi, for me it does not seem to work, even when using the private Vercel solution:

![immagine](https://github.com/anuraghazra/github-readme-stats/assets/18620868/5c31a30e-7f68-42d0-b4b0-4e9d980383a9)
Generated with these options: username=AlessioQuercia&include_all_commits=true&show_icons=true&theme=radical.

![immagine](https://github.com/anuraghazra/github-readme-stats/assets/18620868/a728a6ae-ead4-44f8-b989-daff43187149)

Am I missing something?

**Reply by qwerty541** [reply]
Hey, @AlessioQuercia! Displaying private stats on your card will work only if you have correctly configured permissions of your GitHub API token. Make sure that you have enabled `user` and `repo` permissions as below on screenshots.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/9c94bb06-a59d-402e-8014-5f821046b630)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/c835b969-9850-4cc9-82ed-b9207214b47f)

**Reply by AlessioQuercia** [reply]
Thanks, this solved it!

**Reply by yezhengli-Mr9** [reply]
hi @qwerty541, @AlessioQuercia  How to get to options of the screenshot pages? Thanks~

**Reply by mouragst** [reply]
> hi @qwerty541, @AlessioQuercia How to get to options of the screenshot pages? Thanks~

Go to Settings -> Developer Settings (last option on the left side menu) -> Personal Access Token -> Tokens(classic) -> Generate New Token (classic)

**Reply by qwerty541** [reply]
Here is a link on our recently updated documentation about getting your personal access token (both types classic and fine-grained) https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#first-step-get-your-personal-access-token-pat
